.babelrc

```diff
@@ -1,5 +1,6 @@
 {
   "presets": [
-    "@babel/preset-env"
+    "@babel/preset-env",
+    "@babel/preset-react"
```

Comment:
@kdo does this need `@wordpress/default` as well.  Our last and current project did.... (?)